optimade/server/middleware.py

```diff
@@ -72,6 +75,130 @@ async def dispatch(self, request: Request, call_next):
         return response
 
 
+class HandleApiHint(BaseHTTPMiddleware):
+    """Handle `api_hint` query parameter"""
+
+    @staticmethod
+    def handle_api_hint(api_hint: List[str]) -> Union[None, str]:
+        """Handle `api_hint` parameter"""
+        from optimade.server.routers.utils import BASE_URL_PREFIXES
```

Comment:
Why is this being lazy-loaded inside a static method?

Reply:
The issue is I didn't want the CONFIG to be loaded too early.
But let me just try moving it to the top and see if all goes well.

The middleware is a bit tricky to handle sometimes...